golang/cosmos/x/vstorage/keeper/keeper.go

```diff
@@ -181,6 +181,49 @@ func (k Keeper) ImportStorage(ctx sdk.Context, entries []*types.DataEntry) {
 	}
 }
 
+// RemoveEntriesWithPrefix removes all storage entries starting with the
+// supplied pathPrefix, which may not be empty.
+// It has the same effect as listing children of the prefix and removing each
+// descendant recursively.
```

Comment:
Note that it also creates a no-value entry from the path prefix.

Reply:
It does not. It uses setStorage with a no value to **delete** the entry and any parents